pkg/util/util.go

```diff
@@ -98,3 +128,96 @@ func AddAnnotation(item runtime.Unstructured, annotation, value string) {
 
 	metadata.SetAnnotations(annotations)
 }
+
+func IsVMIPaused(vmi *kvv1.VirtualMachineInstance) bool {
+	for _, c := range vmi.Status.Conditions {
+		if c.Type == kvv1.VirtualMachineInstancePaused {
+			return true
+		}
+	}
+
+	return false
+}
+
+// This is assigned to a variable so it can be replaced by a mock function in tests
+var IsDVExcludedByLabel = func(namespace, dvName string) (bool, error) {
+	client, err := GetCDIclientset()
+	if err != nil {
+		return false, err
+	}
+
+	dv, err := (*client).CdiV1beta1().DataVolumes(namespace).Get(context.TODO(), dvName, metav1.GetOptions{})
+	if err != nil {
+		return false, err
+	}
+
+	labels := dv.GetLabels()
+	if labels == nil {
+		return false, nil
+	}
+
+	label, ok := labels["velero.io/exclude-from-backup"]
```

Comment:
Is there a const for that in velero API? 
if not, then maybe we can have a const in the util

Reply:
Surprisingly, no, velero does not have a const for that. I'll add it in the plugin.